pkg/cmd/cli/describe/describer.go

```diff
@@ -1369,7 +1369,7 @@ func describeBuildTriggerCauses(causes []buildapi.BuildTriggerCause, out *tabwri
 func squashGitInfo(sourceRevision *buildapi.SourceRevision, out *tabwriter.Writer) {
 	if sourceRevision != nil && sourceRevision.Git != nil {
 		rev := sourceRevision.Git
-		formatString(out, "Commit", fmt.Sprintf("%s (%s)", rev.Commit[:7], rev.Message))
+		formatString(out, "Commit", fmt.Sprintf("%s (%s)", rev.Commit, rev.Message))
```

Comment:
that was my first solution, but went with this(i was thinking since we show the entire message...)  . sure, i can do that.